oak-api/src/main/java/org/apache/jackrabbit/oak/api/jmx/QueryEngineSettingsMBean.java

```diff
@@ -86,6 +86,38 @@
 
     void setFastQuerySize(boolean fastQuerySize);
 
+    /**
+     * Whether Path restrictions are enabled while figuring out index plan
+     *
+     * @return true if enabled
+     */
+    String getStrictPathRestriction();
+
+    /**
+     *  Whether path restrictions of indexes (excludedPaths / includedPaths) are taken into account during query execution,
+     *  for Lucene indexes. When enabled, only indexes are considered if the index path restriction is compatible with the
+     *  query path restrictions. When disabled, only the queryPaths of the index is taken into account.
+     *
+     * @param pathRestriction Set path restriction: Expected value is either of ENABLE/DISABLE/WARN
+     *                        ENABLE: enable path restriction- Index won't be used if index definition path restrictions are not compatible with query's path restriction
```

Comment:
The same info is available in "@Description" below... even thought that won't result in Javadocs, I guess it's better to just keep the documentation in "@Description", and simplify the Javadocs. That way, we don't repeat the info (and so only have to update one case).

Reply:
removed @description as it is also available on parameter